JitsiConference.js

```diff
@@ -262,18 +262,28 @@ JitsiConference.prototype.getExternalAuthUrl = function (urlForPopup) {
 JitsiConference.prototype.getLocalTracks = function (mediaType) {
     let tracks = [];
     if (this.rtc) {
-        tracks = this.rtc.localTracks.slice();
-    }
-    if (mediaType !== undefined) {
-        tracks = tracks.filter(
-            (track) => {
-                return track && track.getType && track.getType() === mediaType;
-            });
+        tracks = this.rtc.getLocalTracks(mediaType);
     }
     return tracks;
 };
 
 /**
+ * Obtains local audio track.
+ * @return {JitsiLocalTrack|undefined}
+ */
+JitsiConference.prototype.getLocalAudioTrack = function () {
+    return this.rtc ? this.rtc.getLocalAudioTrack() : undefined;
```

Comment:
I'd say we did not settle at all I just changed to null that particular case. In my opinion it's fine to use whichever as long as it's stated in JSDoc. Otherwise we would have to ask the whole team and come up with a strict rule.